pages/Documents/DeveloperTools/LivePersonFunctions/GettingStarted/monitoring.md

```diff
@@ -118,3 +118,12 @@ All activities will be stored for roughly two months and can be reviewed in this
 Clicking on an individual activity within the activity stream will bring you to a page based on the content. For example, a `Code changed` activity will bring you to a page comparing the previous and that versions. Things like changes to the allowlisting will get you to the allowlist settings.
 
 <img alt="Functions:Activity Context Action" src="img/functions/functions_reporting_activties_context_action.png"> 
+
+### Invocation Histogram
```

Comment:
Please add extra line between heading and paragraph (for consistency with rest of document).

Reply:
(Per internal conversation, deferring this to approve and merge now.)